Mathlib/Analysis/Cauchy.lean

```diff
@@ -0,0 +1,89 @@
+/-
```

Comment:
This file name seems too generic and the location too high up in the tree as well.

Reply:
I'll rename it to `CauchyBound`. What would be a good location?

Reply:
Rename `Mathlib/Analysis/SpecialFunctions/Polynomials` to `.../Polynomial/Basic` and move your file to `.../Polynomial/CauchyBound`?